currency_rate_update_VN_VCB/__manifest__.py

```diff
@@ -0,0 +1,16 @@
+# © 2018 Komit <http://komit-consulting.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    'name': 'Currency Rate Update - VCB',
+    'summary': 'Currency Rate Update Vietcombank service',
+    'version': '11.0.0.0.0',
+    "author": "Komit, Odoo Community Association (OCA)",
+    "website": "http://komit-consulting.com",
+    "license": "AGPL-3",
+    "category": "Financial Management/Configuration",
+    "depends": [
+        "base",
+        "currency_rate_update"
```

Comment:
We should depend on currency_rate_inverted

Reply:
We should force inverted rate for all currencies of the supported_currency_array

Reply:
I custom a little bit model `currency.rate.update.service` to force inverted rate for all currencies of the supported_currency_array. Please check it.